### DISCUSSION TITLE
Licensing: What can and can't we do with Theia?

### DISCUSSION THREAD

**Discussion by trusktr** [eclipse-theia/theia#8478] [General] [answered]
Would someone mind explaining, by giving some practical examples, of what we can and can't do with Theia, and how different licenses apply depending on the use case?

**Comment by marcdumais-work** [comment, marked as answer]
Hi @trusktr

Here are some resources that might help:

https://www.eclipse.org/legal/epl-2.0/faq.php
https://en.wikipedia.org/wiki/Eclipse_Public_License
